util.go

```diff
@@ -50,19 +61,262 @@ func parseTag(path string) string {
 	return strings.TrimSuffix(filepath.Base(path), ".toml")
 }
 
-func parseDependencies(r io.Reader) ([]dependency, error) {
-	var deps []dependency
+func parseDependencies(commit string) ([]dependency, error) {
+	rd, err := fileFromRev(commit, vendorConf)
+	if err == nil {
+		return parseVendorConfDependencies(rd)
+	}
+	rd, err2 := fileFromRev(commit, modulesTxt)
```

Comment:
then after we assign only... and we also don't generally use err, err2, err3.. 

instead we follow the convention of reusing err over and again to catch the last err

Reply:
I figured having all the errors could be useful, as they're looking for different things. Fixed though